sdk/identity/azure-identity/src/main/java/com/azure/identity/AadCredentialBuilderBase.java

```diff
@@ -111,14 +111,18 @@ public T additionallyAllowedTenants(List<String> additionallyAllowedTenants) {
     }
 
     /**
-     * Disables instance discovery.
+     * Disables instance discovery. The functionality to disable instance discovery is limited to Azure Stack
```

Comment:
From team chat, this is the updated doc:
```java
/** Disables the setting which determines whether or not instance discovery is performed when attempting to
* authenticate. This will completely disable both instance discovery and authority validation.
* This functionality is intended for use in scenarios where the metadata endpoint cannot be reached, such as in
* private clouds or Azure Stack. The process of instance discovery entails retrieving authority metadata from
* https://login.microsoft.com/ to validate the authority. By utilizing this API, the validation of the authority
* is disabled. As a result, it is crucial to ensure that the configured authority host is valid and trustworthy.
*/
```